pkg/validation/config.go

```diff
@@ -78,7 +105,9 @@ func validateConfiguration(config *api.ReleaseBuildConfiguration, org, repo stri
 	}
 
 	validationErrors = append(validationErrors, validateReleases("releases", config.Releases, config.ReleaseTagConfiguration != nil)...)
-
+	if config.Images != nil {
```

Comment:
Why does this need a `nil` check when the others (base, rpm) do not?

Reply:
Hmm, that and passing `config.Images` directly got lost in rebases.  Fixed.